### ISSUE TITLE
Make background images available under client control

### ISSUE THREAD

**Issue by avocadesign** [soflyy/oxygen-bugs-and-features#2290]
**Describe the feature you'd like to see included in Oxygen.**
Client control is great for cleaning up the editing experience for clients. However, it has one consistent and major flaw that forces us to turn it off again for quite a lot of users, giving them full access to the design tool and risking them breaking things. That issue is to do with not being able to edit background images. Due to not being able to access the advanced tab, they can't change the background image on a section or div. 

These are almost always "content" rather than "design" on sites we build. 

We want them to be able to access our design library in client control mode and import a section with a nice background image design but then change that image to feature one of their images. Ideally we'd like them to be able to tweak that image position and/or swap to a different image on mobile. 

The easiest way I can think of to do this with client control turned on is to add the background image url field as a feature of the section and div primary panel. If we were wishlisting that we'd love the position controls to also be there. 

I think that's easier than enabling access to certain parts of the advanced tab in client control

**What are the use cases for this feature?**
Clients editing background images without having to ask for help or being given full access to the editor

**Examples of this feature or functionality.**
Most other builders have a way to access the background images and most clients think this is a key requirement.

**Comment by githrun** [comment]
I think the same for images.    I would like to be able to provide access with client control to ONLY:
- text
- images and 
- background images.

And nothing else

**Comment by doddledesign** [comment]
IMO this is the biggest "missing" feature of Oxygen and would be a total game changer as it allows you to completely control the design of each block, whilst giving clients the ability to build pages but in a controlled way. 

To me it makes sense that we should be able to define an ACF field group, use oxygen to control the design, and then allow this block to be added via Guttenberg and set up by the client.
See my post here: https://github.com/soflyy/oxygen-bugs-and-features/issues/2571

**Comment by webSKOR** [comment]
Agreed, this would be very important, because any workaround is very annoying.
